orchestration/dagster_orchestration/hca_orchestration/tests/test_solids.py

```diff
@@ -13,12 +13,13 @@
 
 
 class PostImportValidateTestCase(unittest.TestCase):
+    @patch("hca_manage.manage.HcaManage.get_entities_with_dangling_proj_refs", return_value=set())
     @patch("hca_manage.manage.HcaManage.get_null_filerefs", return_value=set())
     @patch("hca_manage.manage.HcaManage.get_file_table_names", return_value=set())
     @patch("hca_manage.manage.HcaManage.get_duplicates", return_value=set())
     @patch("hca_manage.manage.HcaManage.get_all_table_names", return_value=set())
```

Comment:
probably out of scope for this pr, but this slowly expanding list of patches makes me think we should refactor the `HcaManage` class with a pattern like dependency injection, so we can just slot in a mock database record fetcher and call it a day

Reply:
Yeah, good call: https://broadinstitute.atlassian.net/browse/DSPDC-1662